### ISSUE TITLE
language

### ISSUE THREAD

**Issue by senolfndk** [muzik-apps/Muzik-offline#93]
Can you add Turkish language?

**Comment by waveyboym** [comment]
@senolfndk 
Yes, but you may have to wait a very very long time as this is not an easy or straight-forward feature to just add.
